scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/TaskLauncher.java

```diff
@@ -355,7 +355,7 @@ public void getStoredLogs(AppenderProvider logSink) {
     public PublicKey generatePublicKey() throws NoSuchAlgorithmException {
         KeyPairGenerator keyGen;
         keyGen = KeyPairGenerator.getInstance("RSA");
-        keyGen.initialize(1024, new SecureRandom());
+        keyGen.initialize(2048, new SecureRandom());
```

Comment:
Should we not have a public static constant defined in KeyGen for the key size ?